### PR TITLE
chore: avoid using relative paths

### PR DESCRIPTION
**Pull request by santteegt**
This PR removes relative path usage, so data & templates are always read from the base path

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,15 @@ This dashboard is a tool to collect community opinions on the proper parameteriz
...
 ```
 pip3 install -r requirements.txt
+pip install -e ./
 jupyter labextension install @pyviz/jupyterlab_pyviz
 ```
 
 ## Running Locally
 
 ```
 cd apps
```
Comment (mendesfabio) [inline README.md:21]:
```suggestion
```

It seems we should remove this, right?

File: apps/hatch.py [hunk 2]
```diff
@@ -14,15 +14,13 @@
...
-sys.path.append('..')
-
 from tech.tech import read_impact_hour_data, read_cstk_data, TECH
 from tech.tech import ImpactHoursData, ImpactHoursFormula, Hatch, DandelionVoting
 import tech.config_bounds as config_bounds
 
 load_dotenv()
 
-env = Environment(loader=FileSystemLoader('..'))
+env = Environment(loader=FileSystemLoader('.'))
```
Comment (mendesfabio) [inline apps/hatch.py:23]: Add this to `test-hatch.py` as well.

File: Procfile [hunk 3]
```diff
@@ -1 +1 @@
-web: cd apps && panel serve --address="0.0.0.0" --port=$PORT *.py --allow-websocket-origin=tec-hatch-demo.herokuapp.com --allow-websocket-origin=dev-tec-hatch-demo.herokuapp.com
+web: panel serve --address="0.0.0.0" --port=$PORT apps/hatch.py --allow-websocket-origin=tec-hatch-demo.herokuapp.com --allow-websocket-origin=dev-tec-hatch-demo.herokuapp.com
```
Comment (mendesfabio) [inline Procfile:1]:
```suggestion
web: panel serve --address="0.0.0.0" --port=$PORT apps/*.py --allow-websocket-origin=tec-hatch-demo.herokuapp.com --allow-websocket-origin=dev-tec-hatch-demo.herokuapp.com
```

We want to run both `hatch` and `test-hatch` dashboards.

**Review by mendesfabio — CHANGES_REQUESTED** [review]
thanks, @Santigs! I  left some feedback, please take a look at it. Also, we need to set up a preview deployment to check if everything is working.

**Comment by santteegt** [comment]
@mendesfabio I updated my PR to include your suggestions

**Review by mendesfabio — APPROVED** [review]
LGTM!

**Pull request merged**